meta/runtime.yml

```diff
@@ -1,5 +1,5 @@
 ---
-requires_ansible: '>=2.14.0'
+requires_ansible: '>=2.15.0'
```

Comment:
sanity still failing, see commit for fix: https://github.com/CiscoDevNet/ansible-aci/pull/662/commits/bc0662865516b18da5f8c26f16975843da083904